### DISCUSSION TITLE
Welcome to my first post on  GitHub

### DISCUSSION THREAD

**Discussion by mohamed-fox2** [community/community#145696] [Code Security]
is this real  😂 😑

**Comment by ebndev** [comment]
Hi @mohamed-fox2, thanks for being a part of the GitHub Community!

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" as GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. Any repeat and future violations may result in a temporary or indefinite block from the Community. 

We appreciate your understanding.
